<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSGGlobalsWidget</class>
 <widget class="QWidget" name="SSGGlobalsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSG Globals</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Noise frequency</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="SSGNoiseFreqWidget" name="noiseFreqWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Envelope frequency</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="SSGEnvelopeFreqWidget" name="envFreqWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Envelope shape</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="SSGEnvelopeShapeWidget" name="envShapeWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SSGEnvelopeShapeWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgenvelopeshapewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SSGEnvelopeFreqWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgenvelopefreqwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SSGNoiseFreqWidget</class>
   <extends>QWidget</extends>
   <header>ssg/ssgnoisefreqwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
